src/sections/pages/userDetail/status.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCheck, faExclamation } from '@fortawesome/free-solid-svg-icons';
+import { faClock } from '@fortawesome/free-regular-svg-icons';
+import { Badge } from 'react-rainbow-components';
+
+const iconMap = {
+    VERIFIED: faCheck,
+    EXPIRED: faExclamation,
+    PROCESSING: faClock,
+};
+export default function Status({ value }) {
```

Comment:
this component is twice (in userDetails and orders), put it in components folder and name it OrderStatus